package.json

```diff
@@ -369,7 +368,6 @@
     },
     "comments": {
         "dependencies": {
-            "@hapi/address": "Version 2.1.2+ requires a polyfill for TextEncoder. Pinning to version 2.1.1 prevents IE11 from breaking.",
```

Comment:
Did we test IE11 to ensure this dependency still works as expected?

Reply:
... definitely should have done that. On it now, just waiting for Parallels access and IE11 Win 81 Parallels image to download over country-bumpkin DSL.